scripts/jenkins/cloud/ansible/roles/list_packages/tasks/diff-packages.yml

```diff
@@ -15,23 +15,11 @@
 #
 ---
 
-- name: Diff list of installed packages in plaintext
-  copy:
-    src: "{{ diff_tmp_dir }}/rpms/rpms_{{ item_virtual_hosts }}_{{ state2 }}.txt"
-    dest: "{{ diff_tmp_dir }}/rpms/rpms_{{ item_virtual_hosts }}_{{ state1 }}.txt"
-  loop: "{{ [cloud_env]+groups['cloud_virt_hosts']|flatten(levels=1) }}"
-  delegate_to: localhost
-  check_mode: yes
-  diff: yes
-  register: diff_list
-  loop_control:
-    loop_var: item_virtual_hosts
-
 - name: Create a diff of installed packages and safe into files in plaintext
   shell: |
-    diff -U 0 {{ diff_tmp_dir }}/rpms/rpms_{{ item_virtual_hosts }}_{{ state1 }}.txt \
-      {{ diff_tmp_dir }}/rpms/rpms_{{ item_virtual_hosts }}_{{ state2 }}.txt \
-      > {{ diff_tmp_dir }}/rpms/diffs/rpms_{{ item_virtual_hosts }}.diff
+    diff -U 0 {{ diff_tmp_dir }}/rpms/rpms_{{ item_virtual_hosts }}_{{ state1 }}.yml \
```

Comment:
I know. My intention was to do not change this part much since it wasn't purpose of the card but still think that having some output about differences is valuable when something goes wrong during maintenance update.